rules/table_wrapper.js

```diff
@@ -0,0 +1,72 @@
+/*
+Copyright 2022 DigitalOcean
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+'use strict';
+
+/**
+ * @module rules/table_wrapper
+ */
+
+const safeObject = require('../util/safe_object');
+
+/**
+ * @typedef {Object} TableWrapperOptions
+ * @property {string} [className='table-wrapper'] Class to use for the table wrapper.
+ */
+
+/**
+ * Wrap `tables` with a `div` with `table-wrapper` class.
+ *
+ * @example
+ * <table>...</table>
+ *
+ * <div class="table-wrapper"><table>...</table></div>
```

Comment:
The content here should match what's in the README, please (if you look at some other plugins you should be able to see how the content maps)